src/types.d.ts

```diff
@@ -63,9 +63,9 @@ export interface EnterpriseCustomerUser {
 export interface EnterpriseLearnerData {
   enterpriseCustomer: Types.EnterpriseCustomer | null;
   activeEnterpriseCustomer: Types.EnterpriseCustomer | null;
-  allLinkedEnterpriseCustomerUsers: EnterpriseCustomerUser[];
+  allLinkedEnterpriseCustomerUsers: EnterpriseCustomerUser[] | [];
```

Comment:
The types don't need the fallback; the types is just defining the expected type. `EnterpriseCustomerUser[]` states that its an array of `EnterpriseCustomerUser` objects, even if the array is empty.